builder/bundle_control.go

```diff
@@ -375,10 +375,12 @@ func (b *Builder) AddBundles(bundles []string, allLocal bool, allUpstream bool,
 
 		bundle, err := b.getBundleFromName(bName)
 		if err != nil {
-			return err
+			fmt.Println(err.Error() + "; skipping")
```

Comment:
This looks more like a warning that a normal message, so I believe it would be better to use the log API. Something like `log.Println("Warning: " + err.Error() + "; skipping" )`.

Reply:
@rchiossi Done.